.github/workflows/release.yaml

```
@@ -0,0 +1,31 @@
+name: Release-Tag
+on:
+  push:
+    branches: [ main ] 
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)